enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx

```diff
@@ -0,0 +1,85 @@
+import { type ChangeEvent, useCallback } from "react";
+import { t } from "ttag";
+
+import type {
+  ModelFilterControlsProps,
+  ModelFilterSettings,
+} from "metabase/browse/models";
+import { useUserSetting } from "metabase/common/hooks";
+import { getSetting } from "metabase/selectors/settings";
+import { Button, Icon, Paper, Popover, Switch, Text } from "metabase/ui";
+import type { State } from "metabase-types/store";
+
+const USER_SETTING_KEY = "browse-filter-only-verified-models";
+
+export function getDefaultModelFilters(state: State): ModelFilterSettings {
+  return {
+    verified: getSetting(state, USER_SETTING_KEY) ?? false,
+  };
+}
+
+// This component is similar to the ModelFilterControls component from ./ModelFilterControls.tsx
+// merging them might be a good idea in the future.
+export const ModelFilterControls = ({
+  modelFilters,
+  setModelFilters,
+}: ModelFilterControlsProps) => {
+  const areAnyFiltersActive = Object.values(modelFilters).some(Boolean);
+
+  const [_, setUserSetting] = useUserSetting(USER_SETTING_KEY);
+
+  const handleVerifiedFilterChange = useCallback(
+    function (evt: ChangeEvent<HTMLInputElement>) {
+      setModelFilters({ ...modelFilters, verified: evt.target.checked });
+      setUserSetting(evt.target.checked);
+    },
+    [modelFilters, setModelFilters, setUserSetting],
+  );
+
+  return (
+    <Popover position="bottom-end">
+      <Popover.Target>
+        <Button
+          p="sm"
+          lh={0}
+          variant="subtle"
+          color="text-dark"
```

Comment:
```suggestion
          color="var(--mb-color-text-dark)"
```

Reply:
if you want to update this to our new way, in passing

Reply:
I didn't know it's a new way. It should be safe to use mantine theme colors by default. Anything changed?